package.json

```diff
@@ -12,8 +12,5 @@
     "@commitlint/cli": "19.5.0",
     "@commitlint/config-conventional": "19.5.0",
     "husky": "9.1.6"
-  },
-  "scripts": {
-    "prepare": "husky"
```

Comment:
no conventional commits anymore? 🫨

Reply:
When I tried to commit, I always got some nodejs traceback. It didn't told me what I did wrong, just crashed. Honestly, I didn't see much use in those hooks for this particular project. Since I was anyway forced to skip the commit hooks, I decided to deactivate the automated installer so others don't run into the same problem.

Maybe I'm just ignorant and simply don't get the point. Would you mind explaining what we achieve with husky or conventional commits in this project?

Reply:
Using [conventional commits](https://www.conventionalcommits.org) is used to automatically create [releases for this project](https://github.com/flipdot/drinks-touch/releases) and to fill in the [changelog](https://github.com/flipdot/drinks-touch/blob/master/CHANGELOG.md). Right now those contain the weekly automatic release only as there has been little development going on. So what spending a thought on the commit title lets you benefit is that if you implement a `feat: new component` the project (versioned at 0.4.47) would automatically release a version 0.5.0. Or if you implement a `fix: correct variable name` version 0.4.48 would be released, all without you having to manually go through some release process.

See some insightful example where more frequent development is going on over at: [dargmuesli/nuxt-cookie-control/releases](https://github.com/dargmuesli/nuxt-cookie-control/releases).

It's just nice to never have to think about releases again, just by typing following a conventional format for commit messages.

The weekly automated releases are just some basic strategy to *release* dependency updates that come in by renovate from time to time. If there was some security fix in some dependency it would not take too long to be made available for deployment (and in the perfect world be covered by tests and even deployed for you). Necessity for this project is arguable due to the non-public and manual deployment, so we can remove that to not spam the changelog or turn the frequency down. I just had set it up as I planned to develop more on the "perfect world" approach, for which I haven't found time yet apparently :grin: